code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -194,24 +198,24 @@
 			M.emote("scream")
 		else if(effect <= 4)
 			M.visible_message("<span class='warning'>[M] is all sweaty!</span>")
-			M.bodytemperature += rand(5,30)
-			update_flags |= M.adjustBrainLoss(1, FALSE)
-			update_flags |= M.adjustToxLoss(1, FALSE)
-			M.Stun(4 SECONDS, FALSE)
+			M.bodytemperature += 150
```

Comment:
Crank giving spontaneous combustion now

Reply:
this is this high because body temperature normalises super quickly with the atmos around the mob.